TelezshkaLib/Telezshka.cpp

```diff
@@ -31,20 +31,24 @@ Telezshka::Telezshka(int pinsForWheel1 [6], int pinsForWheel2 [6], int pinsForWh
     Serial.println(_doneTurn);
   #endif
 
+
+  /*****
+  *CRUTCH
+  *****/
   _wheels.at(0) = Wheel(pinsForWheel1);
   _wheels.at(1) = Wheel(pinsForWheel2);
   _wheels.at(2) = Wheel(pinsForWheel3);
   
-
+  const double zero = 0.;
```

Comment:
Hmm... Everyone can see that "0." is zero) Sorry, but it is like that pic: https://pp.userapi.com/c834102/v834102105/1001d7/AAcgyerIT64.jpg
So, you should to give more informative name or only leave a comment about it.